### PR TITLE
Adjust Pgsql with the new benchmark-operator changes

### PR DESCRIPTION
**Pull request by PrasadDesala**
Signed-off-by: Prasad Desala <tdesala@redhat.com>

### CODE REVIEW

**Review by OdedViner — APPROVED** [review]
LGTM

File: ocs_ci/ocs/pgsql.py [hunk 1]
```diff
@@ -428,9 +417,7 @@ def respin_pgsql_app_pod(self):
             pod status
 
         """
-        app_pod_list = get_operator_pods(
-            constants.PGSQL_APP_LABEL, constants.RIPSAW_NAMESPACE
-        )
+        app_pod_list = get_operator_pods(constants.PGSQL_APP_LABEL, constants.BMO_NAME)
```
Comment (Avilir) [inline ocs_ci/ocs/pgsql.py:420]: the BMO_NAME is not define in constants.

Comment (PrasadDesala) [reply ocs_ci/ocs/pgsql.py]: Done.

**Review by tiffanyn108 — APPROVED** [review]
LGTM.

**Pull request merged**